packages/next/client/on-demand-entries-utils.js

```diff
@@ -0,0 +1,53 @@
+/* global window, location */
+
+import fetch from 'unfetch'
+
+let evtSource
+export let currentPage
```

Comment:
Looks like this doesn't need to be exported

Reply:
This is exported for the live binding as explained above.